### PR TITLE
[J11] DDF-4210 Replace Boon in platform-migration and admin-core-appservice

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
This is a Java 11 upgrade PR
It replaces Boon with Gson in platform-migration and admin-core-appservice.

#### Who is reviewing it? 

@blen-desta 
@brjeter 
@jhunzik 
@ethantmanns 

#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl
@figliold
@tbatie

#### How should this be tested?
- Unzip and start DDF (do not run installer)
- execute migration:export
- execute migration:decrypt
- install DDF via profile:install command
- after restart, execute migration:import which will bring you back to pre-install state
- re-install DDF via the UI

#### Any background context you want to provide?
This is a Java11 upgrade PR.

#### What are the relevant tickets?
[DDF-4210](https://codice.atlassian.net/browse/DDF-4210)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: features/admin/src/main/feature/feature.xml [hunk 1]
```diff
@@ -109,6 +112,7 @@
         <bundle>mvn:org.ops4j.base/ops4j-base-util-property/1.4.0</bundle>
         <bundle>mvn:ddf.admin.core/admin-core-appservice/${project.version}</bundle>
         <bundle>mvn:ddf.admin.core/admin-core-impl/${project.version}</bundle>
+        <bundle>mvn:com.google.code.gson/gson/${gson-version}</bundle>
         <bundle>mvn:io.fastjson/boon/${boon.version}</bundle>
```
[the remark this answers is not in the document]
Comment (paouelle) [reply features/admin/src/main/feature/feature.xml]: Not yet as there are still other bundles in this feature that have not been modified.

**Comment by paouelle** [comment]
build now

File: features/admin/src/main/feature/feature.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tbatie) [reply features/admin/src/main/feature/feature.xml]: I think we should move boon over to the features that are using it as a transitive dependency and remove it here.

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
Build now

**Comment by paouelle** [comment]
Build now

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

**Pull request merged**